packages/react/src/themes/base/components/Checkbox/checkboxStyles.ts

```diff
@@ -7,7 +7,14 @@ const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, Ch
     alignItems: 'center',
     position: 'relative',
 
-    display: 'inline-flex',
+    // CSS Grid is polifilled only with latest inline-style-prefixer
```

Comment:
could you, please, provide exact version you are implying here?

Reply:
I created an issue #1686, so it will be tracked. We use `5.0.3`, but it works from `5.1.0`: https://github.com/weserio/inline-style-prefixer/blob/master/CHANGELOG.md#510